### PR TITLE
Implement feature to delete comments on click of a button in html file

### PR DESCRIPTION
**Pull request by haoyxc**
Create button for each comment in html file and on click, removes the comment from the page and from the database.

Alert the user if something goes bad and id is not present in the Post request.

image

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ async function fetchFromData() {
 /**
  * Creates a comment element with the content, name of commenter, and date
```
Comment (EatingW) [inline portfolio/src/main/webapp/script.js:78]: Seems like this comment should be updated

**Review by EatingW — COMMENTED** [review]
LGTM! Just a tiny nit about updating a function JSDoc comment.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (haoyxc) [reply portfolio/src/main/webapp/script.js:78]: good catch! thank yoU!

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -90,11 +91,35 @@ function createCommentElement(comment) {
...
 
+/**
+ * Delete a comment from the Datastore given the id. Alerts the user if something goes wrong and comment cannot be deleted.
+ */
+async function deleteComment(id) {
+  const params = new URLSearchParams();
+  params.append('id', id);
+  const resp = await fetch('/delete-data', {method: 'POST', body: params});
+  if (!resp.ok) {
+    alert("AH YIKES! Cannot delete comment");
```
Comment (fenghaolw) [inline portfolio/src/main/webapp/script.js:114]: Optional: alert is generally not a good user experience :) 

Consider using something like toast/snackbar (https://material.io/develop/web/components/snackbars/) to show the error message to the users.

Comment (haoyxc) [reply portfolio/src/main/webapp/script.js:114]: ohh I'll look into this -- thank you!

**Review by EatingW — APPROVED** [review]

**Pull request merged**